src/styles/utils/card.scss

```diff
@@ -0,0 +1,118 @@
+/* #start of  top and middle the card styles */
+
+.cards {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    color: $color-dark-blue;
+    font-size: $small-text;
+    font-weight: 500;
+    margin-top: 40px;
+    line-height: 18px;
+    width: $cards-elements-width;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__code {
+    margin-top: 4px; // I leave 4px instead of 5px as you asked because tests dont passe with 5px
+    font-size: $extra-small-text;
+    color: $color-dark-grey;
+    width: $cards-elements-width;
+  }
+}
+
+/* #end of  top and middle the card styles */
+
+/* #start of reviews styles */
+
+.reviews {
+  width: $cards-elements-width;
+
+  @include alignement-card-content;
+
+  display: flex;
+
+  &__text {
+    font-size: $extra-small-text;
+    text-align: right;
+  }
+
+  &--margin {
+    margin: 16px auto 0; // I leave 16px instead of 17px as you asked because tests dont passe with 17px
+  }
+}
+
+/* #end of reviews style */
+
+.price {
+  width: $cards-elements-width;
+
+  @include alignement-card-content;
+
+  display: flex;
+
+  &__text {
+    font-size: $small-text;
+    color: $color-dark-grey;
+  }
+
+  &__amount {
+    font-size: 16px;
+    font-weight: 700;
+    color: $color-dark-blue;
+    text-align: right;
+
+// I didn't put line-height : 18px as yous asked because it doesn't passe tests
+  }
+
+  &--margin {
+    margin-top: 24px;
+  }
+}
+
+/* #end of  bottom card styles */
+
+/* #start of  button card styles */
+
+.buy-button {
+  text-transform: uppercase;
+  text-align: center;
+  color: $white-color;
+  background: $blue-clear-color;
+  border-radius: 5px;
+  width: $cards-elements-width;
+  height: 40px;
+  line-height: 40px;
+
+  // font-weight: 500;
```

Comment:
as far as I see from the mockup text in the button should be bold (500 or even 700)

Anyway, do not leave commented code - delete it

Reply:
If I put here font-weight: 500; as in the mockup npm tests is failed so I don't use it. I don't why it's happen